enroll/endpoint.go

```diff
@@ -5,28 +5,65 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/micromdm/micromdm/crypto"
+	boltdepot "github.com/micromdm/scep/depot/bolt"
+
+	"github.com/fullsailor/pkcs7"
 	"github.com/go-kit/kit/endpoint"
 	"github.com/micromdm/mdm"
 )
 
 type Endpoints struct {
-	GetEnrollEndpoint endpoint.Endpoint
+	GetEnrollEndpoint       endpoint.Endpoint
+	OTAEnrollEndpoint       endpoint.Endpoint
+	OTAPhase2Phase3Endpoint endpoint.Endpoint
 }
 
 type depEnrollmentRequest struct {
 	mdm.DEPEnrollmentRequest
 }
 
+// TODO: may overlap at some point with mdm.DEPEnrollmentRequest
```

Comment:
Do we need to create an issue in `micromdm/mdm`? Maybe we can find a more suitable name than `mdm.DEPEnrollmentRequest` which covers both uses. Something like:

```
type EnrollmentRequest struct {
    Serial        *string `plist:"SERIAL"`
}
```
Setting all fields as pointers would also allow using `omitempty` in the tags and checking if nil. 
In either case, I'd merge as is here but open an issue with the mdm package.